### PR TITLE
Patch2

### PR DESCRIPTION
**Pull request by GauravSarkar**
i have deleted the docs file and added a docs folder with a new file .please kindly check and merge.

### CODE REVIEW

**Comment by nate-double-u** [comment]
Thanks for helping with this!

Please see [my comment](https://github.com/cncf/hugo-netlify-starter/issues/41#issuecomment-714618769) on the issue #41 regarding how to fix the the DCO check.

File: content/docs/new [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (nate-double-u) [inline content/docs/new:1]: Could you rename the `content/docs/new` file to `content/docs/_index.md` and add the following copy:

```
---
title: "Docs Index"
description: "index of the docs section"
date: 2020-10-22
toc: true
---

This is index content
```

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Could you rename the `content/docs/new` file to `content/docs/_index.md` and add the following copy:

```
---
title: "Docs Index"
description: "index of the docs section"
date: 2020-10-22
toc: true
---

This is index content
```

**Comment by GauravSarkar** [comment]
I have made the changes but I could not clear DCO sign off, please help

**Comment by GauravSarkar** [comment]
@nate-double-u  the DCO error is not solved but I have made the requested changes.

**Comment by nate-double-u** [comment]
> @nate-double-u the DCO error is not solved but I have made the requested changes.

Thanks for making those changes.

The DCO check requires that _all_ commits are correctly signed. The [check details link](https://github.com/cncf/hugo-netlify-starter/pull/44/checks?check_run_id=1297836208) shows how to fix all the commits.

Once you've done that please squash all your commits into one so there is only one commit to merge into the main branch.


## Some more info on DCO

https://github.com/apps/dco

I noticed that you went through some versions of the sign — I recommend using the git `-s` command to automatically apply the signature:
```
$ git commit -s -m 'This is my commit message'
```

in your case, this would produce:
```
This is my commit message

Signed-off-by: GauravSarkar <60042820+GauravSarkar@users.noreply.github.com>
```

**Comment by GauravSarkar** [comment]
I did what you suggested but I am still getting error.

**Comment by nate-double-u** [comment]
I think that the check is getting a little confused with all the commits -- some of which look correctly signed. Remember that the check requires _all_ the commits to be correctly signed.

I recommend squashing all the commits into one, and signing that one. Here's a [good guide to squashing](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec).

I wanted to make sure I'm giving the right advice, so I've tried it [over on a fork](https://github.com/nate-double-u/cncf-hugo-netlify-starter/pull/3/commits) to test, and it does [pass the DCO check there](https://github.com/nate-double-u/cncf-hugo-netlify-starter/pull/3/checks).

**Comment by nate-double-u** [comment]
Hi @GauravSarkar thanks for your help on this issue. I'm going to close this PR in favour of the corrected PR https://github.com/cncf/hugo-netlify-starter/pull/48.

I hope the info here is helpful for future PRs!

**Comment by GauravSarkar** [comment]
Thanks for helping me out on the issue. Looking forward to work on some other issue in this project.

**Pull request closed without merging**